charts/lagoon-remote/templates/docker-host.deployment.yaml

```diff
@@ -56,18 +56,23 @@ spec:
           value: {{ . | quote }}
         {{- end }}
         - name: DOCKER_HOST
-          value: localhost
+          value: tcp://localhost:2375
```

Comment:
we should allow the passing of a dockerHost.host here, but with this as a fallback